### DISCUSSION TITLE
v4.0.1

### DISCUSSION THREAD

**Discussion by balazsorban44** [nextauthjs/next-auth#3323] [Help]
TODO

<hr /><em>This discussion was created from the release <a href='https://github.com/nextauthjs/next-auth/releases/tag/'>v4.0.1</a>.</em>

**Comment by wivarn** [comment]
Not sure if this is the correct place to post this:
I just upgraded to beta.7 this week and I noticed that `refetchInterval` causes a full page refresh (or at least everything nested under the provider). Is this intentional/unavoidable? In my app, this resets the state of the page such as [dropdown menus.](https://headlessui.dev/react/menu) I might just be using it wrong.

**Reply by balazsorban44** [reply]
I would open a separate discussion with your code
